tests/unit/proxy/test_elasticsearch.py

```diff
@@ -530,6 +533,7 @@ def test_create_document(self, mock_uuid: MagicMock) -> None:
                 'tags': [],
                 'badges': [],
                 'total_usage': 0,
+                'programmatic_descriptions': [],
```

Comment:
I wonder what happens if `programmatic_descriptions` is not provided by Elasticsearch.

Reply:
ok sorry, thought i was on the right track. So currently, the models.table.py has this:
```programmatic_descriptions: List[str] = []```
which means that by default it will be an empty list.

Above, the first Table actually does not specify any programmatic_descriptions. Would this be the way to test no programmatic_descriptions being returned by elastic-search? Would a brand new unit-test be useful here? Let me know what you think.